### ISSUE TITLE
[Warlock] Voidwalker Consume Shadows interrupted while moving

### ISSUE THREAD

**Issue by MilosMisic** [dalaranwow/dalaran-wow#5452]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: If you press Consume Shadows (pet spells) while moving the spell is interrupted

**Expected behaviour**: Voidwalker should stop and cast.

**Steps to reproduce the problem**:

1. Move
2. Let Voidwalker follow you
3. Press Consume Shadows

Proof: https://github.com/TrinityCore/TrinityCore/issues/14054

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
Duplicate of https://github.com/dalaranwow/dalaran-wow/issues/3742 - It will be moved tzo the PTR soonish.
